UI/MainForm.cs

```diff
@@ -493,8 +493,6 @@ void CheckForFirmwareUpdates ()
                 }
             }
 
-            // Connected USB devices that are in bootsel mode get added to the firmware flashing list
```

Comment:
not needed anymore because we are scanning continuosly for new USB devices anyways and those that are currently detected are already discovered.

This is just old code which i did not remove by accident.